models/mesh.go

```diff
@@ -1,8 +1,13 @@
 package models
 
 import (
+	"time"
+
 	corev1 "k8s.io/api/core/v1"
 
+	"crypto/x509"
+	"encoding/pem"
```

Comment:
trivial: the go core libs should go up top in the first group

```suggestion
	"crypto/x509"
	"encoding/pem"
	"time"

	corev1 "k8s.io/api/core/v1"
```